fuzz/cmp.c

```diff
@@ -11,19 +11,21 @@
  * Test CMP DER parsing.
  */
 
+/* We need to use some deprecated APIs */
```

Comment:
Why?? cmp is new for 3.0 we really shouldn't be using deprecated stuff.

Reply:
There might have been a reason but seeing as it can be removed, I'm not going to go looking.